aten/src/ATen/core/ivalue.cpp

```diff
@@ -1034,7 +1034,7 @@ TORCH_API intrusive_ptr<ivalue::Future> collectAny(
       if (src->hasError()) {
         dst->setError(src->exception_ptr());
       } else {
-        dst->markCompleted(src->constValue());
+        dst->markCompleted(src->constValue(), src->dataPtrs());
```

Comment:
Is it because collectAll's dst future is a join instead of child, so we cannot pass `dataPtrs` there? If so, shall we add a brief comment to explain that?

Reply:
NVM, I just saw this is already added in the next PR.